utils/spdx/src/main/kotlin/model/SpdxPackage.kt

```diff
@@ -66,7 +66,7 @@ data class SpdxPackage(
      * A text relating to a copyright notice, even if not complete. To represent a not present value
      * [SpdxConstants.NONE] or [SpdxConstants.NOASSERTION] must be used.
      */
-    val copyrightText: String,
+    val copyrightText: String = SpdxConstants.NOASSERTION,
```

Comment:
Have you considered to omit "default" values from serialization?

Reply:
I haven't considered that TBH, but looking at other non-empty / non-null default values (like for `licenseInfoInFiles`) we seem to don't exclude these, either. And given the unclarity of the 2.2 specs, it's maybe better to be explicit. Do you agree?

Reply:
As-is when reading a file and writing it, omited values in the input will be filled with default values in the output. I considered this as a slightly negative thing. While at the same time the "require not blank" condition in the `init` block already guarantees that we are actually explicit in the sense that one has to specify to the constructor the value. It's not explicitly written out.

Well, I don't have a strong opinion here, actually I don't know which one is better. You can make the call.